crates/re_arrow_store/tests/data_store.rs

```diff
@@ -75,18 +75,16 @@ fn all_components() {
         let cluster_key = store.cluster_key();
 
         let components_a = &[
-            ColorRGBA::name(),           // added by us, timeless
-            Rect2D::name(),              // added by us
-            cluster_key,                 // always here
-            re_log_types::MsgId::name(), // injected automatically (..for now)
+            ColorRGBA::name(), // added by us, timeless
+            Rect2D::name(),    // added by us
```

Comment:
"us" is quite ambiguous imho. I first read it as "Added by data store, not by the user" which is the opposite of what it means

```suggestion
            ColorRGBA::name(), // added by test, timeless
            Rect2D::name(),    // added by test
```